plugins/net_plugin/net_plugin.cpp

```diff
@@ -527,9 +526,9 @@ namespace eosio {
       struct queued_write;
       void fill_out_buffer( std::vector<boost::asio::const_buffer>& bufs,
                             deque<queued_write>& w_queue ) {
-         while ( w_queue.size() > 0 ) {
+         while ( !w_queue.empty() ) {
             auto& m = w_queue.front();
-            bufs.push_back( boost::asio::buffer( *m.buff ));
+            bufs.emplace_back( boost::asio::buffer( *m.buff ));
```

Comment:
Should it be `boost::asio::const_buffer( *m.buff )`? Also why emplace_back?

Reply:
`boost::asio::const_buffer` didn't compile. `emplace_back` since it is calling a constructor with `boost::asio::buffer`

Reply:
This works: `bufs.push_back( boost::asio::const_buffer( m.buff->data(), m.buff->size() ));`

Reply:
I think even better: `bufs.emplace_back( m.buff->data(), m.buff->size() );`